sys-process/runit/files/1-2.1.2

```diff
@@ -1,10 +1,11 @@
-#!/bin/sh
+#!/bin/bash
 # system one time tasks
 
+. /etc/runit/rc.sh
```

Comment:
It looks inconsistent to set PATH before loading `/etc/runit/rc.sh` in `3` script but set PATH after loading `/etc/runit/rc.sh` in `1` script. Probably it's a mistake here and PATH should be set before loading `rc.sh`.